src/transformers/modeling_utils.py

```diff
@@ -2526,7 +2535,9 @@ def save_pretrained(
 
         hf_quantizer = getattr(self, "hf_quantizer", None)
         quantization_serializable = (
-            hf_quantizer is not None and isinstance(hf_quantizer, HfQuantizer) and hf_quantizer.is_serializable
+            hf_quantizer is not None
+            and isinstance(hf_quantizer, HfQuantizer)
+            and hf_quantizer.is_serializable(safe_serialization)
```

Comment:
As this is an optional arg, I prefer to pass it like that.
```suggestion
            and hf_quantizer.is_serializable(safe_serialization=safe_serialization)
```